stub.php

```diff
@@ -8,14 +8,14 @@
     @ini_set('opcache.enable_cli', '0');
 
     if (isset($_GET['opcache_reset']) && $_GET['opcache_reset'] === md5(Phar::running(false))) {
-        $opcacheEnabled = @ini_get('opcache.enable');
+        $GLOBALS['opcacheEnabled'] = @ini_get('opcache.enable');
```

Comment:
why would this write to `$GLOBALS`?

Reply:
To fix the `UnusedVariableSniff` error. 
But as the stub file is no longer included in the ecs config, I can revert that if you want?

_(Although I think it is better to understand this way because the intent of this being global is way more clear)_